.circleci/config.yml

```diff
@@ -111,18 +111,15 @@ jobs:
             - bin/
   release:
     environment:
-        CIRCLE_PROJECT_USERNAME: cloudskiff
-        CIRCLE_PROJECT_REPONAME: driftctl
+      CI: circleci
     docker:
-      - image: cibuilds/github:0.13
+      - image: golang:1.16
     steps:
       - attach_workspace:
```

Comment:
This is now useless, it was previously used to share build artifact between jobs, but since we build, sign and publish in a single job this now useless

Reply:
Isn't it useful to publish on AUR ?

Reply:
AUR only need the SHA256SUM file, now that is is generated here we need to add a persist step after the build.

```yaml
      - persist_to_workspace:
          root: ~/project
          paths:
            - bin/driftctl_SHA256SUMS
```